ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -439,7 +454,22 @@ public int FindTargetForTouch(int reactTag, double touchX, double touchY)
             }
 
             var uiElement = view.As<UIElement>();
+#if WINDOWS_UWP
             var target = VisualTreeHelper.FindElementsInHostCoordinates(new Point(touchX, touchY), uiElement)
+#else
+            var sources = new List<DependencyObject>();
+            VisualTreeHelper.HitTest(
```

Comment:
Honestly, since you're doing FirstOrDefault() below anyway, I would separate the UWP and .NET 4.6 behaviors a bit more.  With the HitTest approach, you can inline the HasTag() test in the HitTest delegate body.

---
In reply to: [85234608](https://github.com/ReactWindows/react-native-windows/pull/831#discussion_r85234608) [](ancestors = 85234608)